templates/partials/taxonomylist.html.twig

```diff
@@ -1,10 +1,13 @@
 {% set taxlist = children_only is defined ? taxonomylist.getChildPagesTags() : taxonomylist.get() %}
+{% set limit = config.plugins.taxonomylist.limit %}
 
 {% if taxlist %}
 <span class="tags">
-    {% for tax,value in taxlist[taxonomy] %}
-        {% set active = uri.param(taxonomy) == tax? 'active' : '' %}
-        <a class="{{ active }}" href="{{ base_url }}/{{ taxonomy }}{{ config.system.param_sep }}{{ tax }}">{{ tax }}</a>
+    {% set count = 0 %}
+    {% for tax,value in taxlist[taxonomy] if count < limit %}
+        {% set label_class = uri.param(taxonomy) == tax ? 'label-primary' : 'label-secondary' %}
```

Comment:
This also changes class names. Not only the logic.

Reply:
I didn't touch anything, just add a condition in the loop which refers to a variable in the taxonomy.yaml, because I noticed that the plugin already put the tags in order from the most used one to the less used ones.

Reply:
Well, the `{% set active` line has been removed, and you have introduced `label-primary` and `label-secondary` to the code. Your contribution is probably not originating from the plugin itself, but from a customized (themed?) twig.

Reply:
You are right, I forgot about my old modifications before posting the code.
I have to admit it: that's my first time posting something on github.
I'll amend the lines soon.